docs/app/components/select/select-demo.component.ts

```diff
@@ -3,15 +3,12 @@ import { Component, ViewEncapsulation } from '@angular/core';
 @Component({
     selector: 'hc-select-demo',
     templateUrl: './select-demo.component.html',
-    styleUrls: [
-        './select-demo.component.scss'
-    ]
+    styleUrls: ['./select-demo.component.scss']
 })
+
 export class SelectDemoComponent {
-    showTemplate: boolean = true;
+    validCheck: boolean = true;
```

Comment:
I would remove `validCheck` and `toggleValidate()` and rely on angular forms functionality instead.